src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -10983,13 +10983,18 @@ static bool isCandidateUnique(ref MethodSymbol? method, MethodSymbol candidate)
             useParams = false;
             MethodSymbol? foundMethod = null;
             var typeArguments = node.TypeArgumentsOpt;
+            int arity = typeArguments.IsDefaultOrEmpty ? 0 : typeArguments.Length;
+
+            // 1. instance methods
             if (node.ResultKind == LookupResultKind.Viable)
             {
                 var methods = ArrayBuilder<MethodSymbol>.GetInstance(capacity: node.Methods.Length);
                 foreach (var memberMethod in node.Methods)
                 {
                     switch (node.ReceiverOpt)
                     {
+                        case BoundTypeOrValueExpression:
```

Comment:
:memo: Fixed a bug here. The relevant test is `FunctionType_ColorColorReceiver_04`.